pipeline/train/configs/opustrainer/teacher.one-stage.yml

```diff
@@ -0,0 +1,39 @@
+datasets:
+  original: <dataset0> # Original parallel corpus
+  backtranslated: <dataset1> # Back-translated data
+
+stages:
+  - train
+
+# Train on a mix until early stopping
+# (useful for clean back-translated data produced by a strong model)
+train:
+  - original 0.7
+  - backtranslated 0.3
+  - until original inf
+
+# The default values of the modifiers are taken from the paper https://arxiv.org/pdf/2311.14838.pdf
+# Please refer to docs/opus-trainer.md for further details
+modifiers:
```

Comment:
Suggestion (optional): The configs have some duplicated information, it would be a bit more maintainable to have 3 files here, one-stage, two-stage, and a shared config with things like the modifiers seed, etc.

Reply:
I agree but I'll leave this one for the refactoring of the training script. The final config will likely be composed dynamically from multiple configs. I'll file an issue for the refactoring.